adafruit_vl6180x.py

```diff
@@ -26,12 +26,15 @@
 * Adafruit's Bus Device library: https://github.com/adafruit/Adafruit_CircuitPython_BusDevice
 """
 import struct
+import time
+
 from micropython import const
 
 from adafruit_bus_device import i2c_device
 
 try:
-    import typing  # pylint: disable=unused-import
+    import logging
```

Comment:
Is there a reason you chose to use the `logging` module instead of `print()` statements?  As is, this would now require the use of the `adafruit_logging` when used on a microcontroller (or `logging` with Blinka) where it didn't before.

Reply:
Force of habit, to be honest. In the projects I typically work on, `print`ing is a big no-no. 
I have now checked the other CircuitPython repositories for style reference and replaced the two occurences of `logging` with a `print()`. There should be no need to use `adafruit_logging`.